packages/core-data/package.json

```diff
@@ -24,6 +24,7 @@
 		"@babel/runtime": "^7.0.0",
 		"@wordpress/api-fetch": "file:../api-fetch",
 		"@wordpress/data": "file:../data",
+		"@wordpress/deprecated": "file:../deprecated",
```

Comment:
There should have been a corresponding change to the root `package-lock.json`. I'm not sure how this passed the build task which checks for local uncommitted changes, but it shouldn't have.

Reply:
Huh! Looks like `check-local-changes` doesn't pick this up because `precheck-local-changes` doesn't run `npm install`.

Will fix this up in a seperate PR.

Reply:
https://github.com/WordPress/gutenberg/pull/13462